src/Payments.Jobs.AutoApprove/Program.cs

```diff
@@ -98,13 +136,16 @@ private static ServiceProvider ConfigureServices()
 
             // options files
             services.Configure<FinanceSettings>(Configuration.GetSection("Finance"));
+            services.Configure<SparkpostSettings>(Configuration.GetSection("Sparkpost"));
 
 
             // db service
             services.AddDbContext<ApplicationDbContext>(options =>
                 options.UseSqlServer(Configuration.GetConnectionString("DefaultConnection"))
             );
 
+            services.AddTransient<IEmailService, SparkpostEmailService>();
+            services.AddMjmlServices();
```

Comment:
Ugh, that's right! This is the old mjml system that spins up a node process in the background. I'm going to have to keep an eye on that when upgrading to net10.0.